app/views/content/life-as-a-teacher/age-groups-and-specialisms/primary/_article.html.erb

```diff
@@ -94,6 +94,9 @@
     <h3>Progression as a primary school teacher<h3>
       <p>As a primary school teacher, there's plenty of opportunity for progression and opportunities to lead beyond the classroom. That could mean becoming a lead in a subject, leading a key stage or even becoming a part of the senior management team.</p>
      <%= render 'content/shared/quotes/quote_primary_phil' %>
+    <h3>Find primary teacher training courses<h3>
+      <p>Explore the different primary courses available.</p>
```

Comment:
This feels a bit sparse - we are saying the same thing in the h3, p and link :-D 

Would this be the point to talk about primary with specialisms?